pox-locking/src/events.rs

```diff
@@ -335,11 +341,31 @@ fn create_event_info_data_code(function_name: &str, args: &[Value]) -> String {
                 pox_addr = &args[3],
             )
         }
-        _ => "{{ data: {{ unimplemented: true }} }}".into(),
+        "revoke-delegate-stx" => {
+            if let Value::Optional(opt) = *response.data.clone() {
+                format!(
+                    r#"
+                    {{
+                        data: {{ delegate-to: '{delegate_to} }}
+                    }}
+                    "#,
+                    delegate_to = opt
+                        .data
+                        .map(|boxed_value| *boxed_value)
+                        .unwrap()
+                        .expect_tuple()
+                        .get("delegated-to")
+                        .unwrap()
+                )
+            } else {
+                "{data: {unimplemented: true}}".into()
```

Comment:
Is this reachable?  Same for below.

Reply:
Yes, for older pox contracts, because the older contracts return no data.